### ISSUE TITLE
[Suggestion] You only need to know how many shells there are

### ISSUE THREAD

**Issue by LazyHatGuy** [xplanthris/SimpleBuckshotRouletteCounter#2]
The only input for a new round should be shell count because you can predict everything else
If there is an odd number of shells, there are always more blank shells than live shells

[Example in another helper](https://ljovynn.github.io/buckshot/main.html)

**Comment by xplanthris-zz** [comment]
Let me take a look at the other helper, and I'll get back to you

**Comment by xplanthris-zz** [comment]
Yeah I will implement this, thanks for the suggestion

**Comment by xplanthris-zz** [comment]
Implemented in 3b18384, the code for that was really simple.

```
if (total_bullets % 2) == 0:
    blank = total_bullets // 2
    live = total_bullets // 2
else:
    blank = (total_bullets // 2) + 1 
    live = total_bullets // 2
```

Thanks!
